libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -378,6 +379,7 @@ private void initializeView() {
     bind();
     initializeNavigationViewModel();
     initializeSummaryBottomSheet();
+    initializeInstructionListListener();
```

Comment:
I know it would be incredibly hard to cause this crash, but let's initialize this listener after the presenter/event dispatcher because they are being used within the listener